internal/fusefrontend/fs_dir.go

```diff
@@ -24,22 +24,22 @@ import (
 
 const dsStoreName = ".DS_Store"
 
-func (fs *FS) mkdirWithIv(cPath string, mode uint32) error {
+func (fs *FS) mkdirWithIv(dirfd *os.File, cName string, mode uint32) error {
```

Comment:
Needs a comment explaining the arguments

Reply:
OK